.github/workflows/fal_workflow.yml

```diff
@@ -1,24 +1,30 @@
 name: Run fal scripts
 
 on:
-  push:
-    branches: [release]
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
should we run it on every PR? Might get expensive in GCP.

Reply:
What no workflow at all? I got confused with fal_dbt_test.

Reply:
Or just on dispatch?

Reply:
Just dispatch should be good. This is an example of incorporating fal into CI/CD. We're going to have a blog post about this.